### PR TITLE
chore: [M3-5751] - Improve the `README.md`

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Rebrands to Akamai
- Makes our `README.md` look less crusty
- Uses badges to give us (and users) some useful stats
  - CI status (we can't do cypress with our current setup, sorry)
  - Online status for cloud.linode.com
  - Cloud Manager's version
  - Storybook Link
  - api-v4 and validation versions
  - api-v4 and validation bundle sizes
- ~Adds some emojis to add some color (we can remove if it's too cringe)~

## Preview 📷

Screenshot 2023-04-07 at 1 53 30 PM

### CODE REVIEW

**Review by cpathipa — APPROVED** [review]

**Review by mjac0bs — COMMENTED** [review]
Thanks for doing this, @bnussman-akamai! A couple comments:

1. As much as I am a fan of emojis, I lean towards it being a little much for the readme.
2. If we can't get CI status with Cypress, can we remove it for now? I don't see much point in including something that's always going to show 'failing', unless I'm misunderstanding something.
3. Didn't we have a ticket somewhere in the backlog for updating our badges on this readme? We could include that ticket number in this PR. I can't seem to find it right off in our backlog.

**Comment by bnussman-akamai** [comment]
@mjac0bs 

1. Anyone else agree with @mjac0bs , I could go either way
2. the `ci` badge represents the `ci` Github Actions workflow, not Cypress. So basiclly if typechecks and unit tests pass on develop, the badge will show `passing`

**Comment by mjac0bs** [comment]
> the ci badge represents the ci Github Actions workflow, not Cypress. So basically if type checks and unit tests pass on develop, the badge will show passing

👍 Thanks for clarifying! I misread "CI status (we can't do cypress with our current setup, sorry)".

File: README.md [hunk 1]
```diff
@@ -1,29 +1,55 @@
...
+    <img alt="validation bundle size" src="https://img.shields.io/bundlephobia/min/@linode/validation?label=validation size">
+  </a>
+  
 </p>
 
-This repository is home to the [Linode Cloud Manager](https://cloud.linode.com) and related projects, including the [JavaScript SDK](https://www.npmjs.com/package/@linode/api-v4).
+## Overview 📒
 
-## Running Projects Within the Repository
+This repository is home to the Linode **[Cloud Manager](https://cloud.linode.com)** and Linode's related [`@linode/api-v4`](packages/api-v4/) and [`@linode/validation`](packages/validation/) Typescript packages.
```
Comment (dwiley-akamai) [inline README.md:40]: Given the rebranded title above, would we want to say "Akamai Connected Cloud" here too?

**Review by dwiley-akamai — COMMENTED** [review]
I'm in favor of removing the emojis

**Comment by bnussman-akamai** [comment]
Emojis have been removed 🚫

**Review by mjac0bs — APPROVED** [review]
It will be nice to be able to refer to the badges for the current version numbers of our packages when doing things like a release!

Also, I'd consider this PR docs update rather than a chore.

**Review by dwiley-akamai — APPROVED** [review]

**Pull request merged**